staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -157,9 +157,11 @@ func (ag *Config) WatchUpdate() *bcast.Member {
 
 func (ag *Config) ToString() string {
 	text := ""
+	ag.mux.RLock()
```

Comment:
Is it added for the unit test TestInClusterConfig ?

Reply:
This function is currently used only in UT. Just add lock for general purpose. It can be used by other code in the future.